components/content-blocks/Video/index.js

```diff
@@ -1,8 +1,12 @@
 import PropTypes from "prop-types";
 import Container from "@/layout/Container";
 import { Video, Figure } from "@/components/atomic";
+import { useIsMounted } from "@/hooks";
 
 export default function VideoBlock({ caption, url }) {
+  const isMounted = useIsMounted();
+  if (!isMounted) return null;
+
```

Comment:
Was able to quickly track down the hydration error to this video component.  Some conflicting reports as to whether or not a third party iframe _should_ throw a hydration error, but felt like not too much harm in adding this quick and easy check, using an existing hook.  Hook returns true when a component is "mounted" aka the first time `useEffect` fires.  And, as it happens, that only happens browserside, so, for us, it doubles as a cheap check of whether or not we're in the browser.  And if the Video/iframe only loads in browser then React sees no diff in DOM and VDom.